schedule/yast/maintenance/default_sle15sp3.yaml

```diff
@@ -0,0 +1,53 @@
+---
+# Default ordered sequence of steps for sle15sp3 maintenance update
+bootloader:
+  - installation/bootloader_start
+setup_libyui:
+  - installation/setup_libyui
+product_selection:
+  - installation/product_selection/install_SLES
+license_agreement:
+  - installation/licensing/accept_license
+registration:
+  - installation/add_on_product_installation/add_additional_products
+  - installation/registration/register_via_scc
+extension_module_selection:
+  - installation/module_registration/register_extensions_and_modules
+add_on_product:
+  - installation/add_on_product/skip_install_addons
+add_on_product_installation: []
```

Comment:
we should remove all the hooks that we don't use, better be conservative, later on we can add them if needed, we don't know how much we will extend test coverage. so we can remove:
`add_on_product_installation`, `guided_*`, `default_systemd_target`, `security`, `virsh_boot_hdd`, `reconnect_svirt`,